superset/assets/src/visualizations/FilterBox/FilterBox.jsx

```diff
@@ -40,9 +43,13 @@ const TIME_FILTER_MAP = {
   granularity: '__granularity',
 };
 
-const TIME_RANGE = '__time_range';
+export const TIME_RANGE = '__time_range';
+export const FILTER_LABELS = {
+  [TIME_RANGE]: 'Time range',
```

Comment:
should this be wrapped in `t()`?

Reply:
we don't translate values in map. the place where used this map will translate the value.